netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -34,9 +34,11 @@
 import com.google.common.base.Preconditions;
 
 import io.grpc.AbstractChannelBuilder;
+import io.grpc.internal.AbstractReferenceCounted;
 import io.grpc.internal.ClientTransport;
 import io.grpc.internal.ClientTransportFactory;
 import io.grpc.internal.SharedResourceHolder;
+
```

Comment:
Please fix your autoformatter. io.grpc and io.netty are supposed to be in the same group. You don't need to fix the occurrences in this PR, but please fix your IDE.